src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed/DefaultOmniSharpProjectSnapshotManagerAccessor.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
```

Comment:
I wonder if changing the null-ability in here was a good idea. I sort of viewed it all as part of prepping the area, and if  I change it AFTERWARD I'll have to change a ton in the .txt files anyway. But if people want to tell me this should be a separate PR I won't fight too much.

Reply:
I'd prefer separate. Would love if this sort of PR had no real code diff to look at, since it's big, so easier to scan through and make sure only visibility modifiers change